include/boost/beast/_experimental/core/ssl_stream.hpp

```diff
@@ -104,8 +104,8 @@ class ssl_stream
     ssl_stream(
         Arg&& arg,
         boost::asio::ssl::context& ctx)
-        : p_(new stream_type{
-            std::forward<Arg>(arg), ctx})
+        : p_(std::make_unique<stream_type>(
```

Comment:
This is available in C++14 and later, while Beast only requires C++11

Reply:
Oops, correct, I forgot it was a 14 and up feature.